go.mod

```diff
@@ -9,14 +9,14 @@ require (
 	github.com/klauspost/compress v1.15.4
 	github.com/ulikunitz/xz v0.5.10
 	go.starlark.net v0.0.0-20220328144851-d1966c6b9fcd
-	golang.org/x/crypto v0.0.0-20190308221718-c2843e01d9a2
-	golang.org/x/term v0.12.0
+	golang.org/x/crypto v0.21.0
+	golang.org/x/term v0.18.0
```

Comment:
This upgrade is out of scope of this PR.

Reply:
`golang.org/x/term` is required for `golang.org/x/crypto` actually: https://cs.opensource.google/go/x/crypto/+/master:go.mod.

```go
module golang.org/x/crypto

go 1.18

require (
	golang.org/x/net v0.21.0 // tagx:ignore
	golang.org/x/sys v0.19.0
	golang.org/x/term v0.19.0
)

require golang.org/x/text v0.14.0 // indirect
```